ur_robot_driver/launch/ur_control.launch

```diff
@@ -9,6 +9,8 @@
   <arg name="use_tool_communication" doc="On e-Series robots tool communication can be enabled with this argument"/>
   <arg name="controller_config_file" doc="Config file used for defining the ROS-Control controllers."/>
   <arg name="robot_ip" doc="IP address by which the robot can be reached."/>
+  <arg name="reverse_port" doc=""/>
+  <arg name="script_sender_port" doc=""/>
```

Comment:
I think, we can take the same docstrings as we have in the code.

I think default values also would make sense here.